pages/your_tasks/ethics.md

```diff
@@ -0,0 +1,116 @@
+​
+* Ethical issues arise most often in research on or involving humans affecting human dignity and autonomy. These issues are partly addressed by the [General Data Protection Regulation](https://gdprinfo.eu/) (see also the [RDMkit data protection page](data_protection))
+There are additional considerations connected to health research. Under the viewpoint of RDM you should especially consider:
+    * Free and informed consent, the possibility of its withdrawal, its documentation, withdrawal (connected with data deletion) and its representation in a machine-actionable way (see also [What information must be given individuals whose data is collected](https://commission.europa.eu/law/law-topic/data-protection/reform/rules-business-and-organisations/principles-gdpr/what-information-must-be-given-individuals-whose-data-collected_en), the [EU clinical trials regulation](https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=celex%3A32014R0536) and respective national health research legislations)
+    * Balancing access to research data and privacy of research participants (see [How much data can be collected](https://commission.europa.eu/law/law-topic/data-protection/reform/rules-business-and-organisations/principles-gdpr/how-much-data-can-be-collected_en), the [EU clinical trials regulation](https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=celex%3A32014R0536) and respective national health research legislations)
+    * Unintentional discrimination of research participants
+    * Findings about participants that are outside of the original research question(s): see [Purpose of data processing](https://commission.europa.eu/law/law-topic/data-protection/reform/rules-business-and-organisations/principles-gdpr/purpose-data-processing_en) and [Incidental findings Archives - BBMRI-ERIC](https://www.bbmri-eric.eu/elsi-topic/incidental-findings/)
+​
+* Other ethical questions are arising from the impact of research outcomes, including data on the interest of communities or individuals
+    * Fair management of intellectual property rights - also see e.g. ABS
+    * Publication of research data that might impact the reputation of communities or individuals
+    * Publication of research data that might impact economical interests of  communities or individuals
+    * Publication of research data that might impact security of society, communities or individuals
+​
+* General research ethics considerations that are also reflecting on research data, including:
+    * Examination of what data is excluded or included and of the circumstances of its collection
+    * Accuracy and trustworthiness of data sources
```

Comment:
On the risk of being polemic - Accuracy and trustworthy are seldom absolutes and may differ across contexts. :) Maybe it could be framed as - accurate or trustworthy enough for your needs?